.gitignore

```diff
@@ -31,7 +31,6 @@ htmlcov/
 build
 dist
 MANIFEST
-cylc.egg-info
```

Comment:
I finally found out why `pip` was reporting `cylc-8.0b1.dev0` and `cylc-flow-8.0rc1.dev0` in the same environment (and why I couldn't uninstall it). I had a leftover egg file which `git clean` wouldn't remove because it was in the .gitignore.